net-misc/i2pd/i2pd-2.6.0-r3.ebuild

```diff
@@ -28,8 +28,8 @@ DEPEND="${RDEPEND}
 	i2p-hardening? ( >=sys-devel/gcc-4.7 )
 	|| ( >=sys-devel/gcc-4.7 >=sys-devel/clang-3.3 )"
 
-I2PD_USER="${I2PD_USER:-i2pd}"
-I2PD_GROUP="${I2PD_GROUP:-i2pd}"
+I2PD_USER=i2pd
+I2PD_GROUP=i2pd
 
```

Comment:
Is there a reason why we want to prevent the user and group from being overridden by the env vars I2PD_USER and I2PD_GROUP?